account/src/androidMain/kotlin/com/privateinternetaccess/account/internals/AccountHttpClient.kt

```diff
@@ -41,15 +41,16 @@ internal actual object AccountHttpClient {
 
     actual fun client(
         certificate: String?,
-        pinnedEndpoint: Pair<String, String>?
+        pinnedEndpoint: Pair<String, String>?,
+        requestTimeoutMillis: Long?
```

Comment:
you can do here `requestTimeoutMillis: Long = Account.REQUEST_TIMEOUT_MS` which means that if not passed when invoked, it will use `Account.REQUEST_TIMEOUT_MS`. That way you avoid making it nullable.

Reply:
If we remove the nullability from `requestTimeoutMillis` then we also need to update all the other requests that call `client()` and this would be quite a large change 😵:

![image](https://github.com/pia-foss/mobile-shared-account/assets/108072074/75fb059f-10f8-4dfe-ba28-63cf3bf382cb)

Reply:
Please ignore my comment above. Expect a new PR soon 😅

Reply:
did you add the ` = Account.REQUEST_TIMEOUT_MS` on the right side of it to indicate the default value? Let me clone